### PR TITLE
feature: Add project id parameterized commands

### PR DESCRIPTION
**Pull request by mkarrmann**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,136 +1,158 @@
...
-    "typescript": "^3.7.5",
-    "vscode-test": "^1.3.0"
-  },
-  "resolutions": {
-    "minimist": "^1.2.5"
-  },
-  "dependencies": {
-    "@doist/todoist-api-typescript": "1.5.0"
-  }
+	"name": "todoist",
```
Comment (mkarrmann) [inline package.json]: I'll undo these autoformatter changes

**Comment by mkarrmann** [comment]
@waymondo Honestly I wasn't aware you can create a PR from forks :) Here's my changes

**Comment by waymondo** [comment]
Thanks for your patience, this looks good to me. I haven't tested it personally but it looks safe to merge by my eyes. Do you mind just updating the README with information about the new commands?

I would also be open to adding keybindings for these. Is it possible to assign them to <kbd>alt+t i c</kbd> <kbd>alt+t i o</kbd> and <kbd>alt+t i t</kbd>? I'm not sure how defining multiple prefixes work in VSCode though. Also open to an alternative format if you have an idea.

**Comment by mkarrmann** [comment]
@waymondo Thanks! I'll update the README.

I'm not sure about adding default keybindings to these commands- there doesn't seem to be a good way to create a keybinding that accepts an argument. The best I can find is the keybinding opens an inputbox to input the argument. Plus, it seems rather inconvenient to need to remember the project id for a project in order to use the command. 

I'm using this change by creating my own shortcut, and I specify the project id in the shortcut itself. My opinion is that that's the only way to use these new commands that's convenient, and there's no reasonable default behavior that we could specify by creating a default shortcut. Let me know your thoughts!

**Comment by waymondo** [comment]
That's fine with me. Just a little description in the README would suffice then.

**Comment by mkarrmann** [comment]
@waymondo Just pushed the README changes.

In addition to adding sections for the new commands, I had to reword some other parts of the documentation. Truthfully I found it a bit tricky to get the right wording, so I'd love to hear your thoughts.

**Comment by waymondo** [comment]
looks good to me 👍

**Pull request merged**